CONTRIBUTING_MODELS.md

```diff
@@ -72,168 +119,174 @@ contribution! 🙂
 
 #### Add the backbone class
 
-Once you are done identifying all the required layers, you should implement the
-model backbone class.
+Once you've identified the required layers, implement the backbone using [Keras’ functional API](https://keras.io/guides/functional_api/) wrapped in a class.
+
+Compare your code with [`DistilBertBackbone`](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/distil_bert/distil_bert_backbone.py) for structure.
+
+##### Implementation
+
+- Use standard inputs (`token_ids`, `padding_mask`, `pixel_values`, `audio_features`)
+- Use `keras.layers` and `keras_nlp.layers` when possible
+- For architectural deviations, implement custom layers
+
+Examples:
+- Major changes: [`DebertaV3`](https://github.com/keras-team/keras-hub/tree/master/keras_hub/models/deberta_v3)
+- Minor tweaks: [Whisper attention layer](https://github.com/keras-team/keras-hub/pull/801/files#diff-8533ae3a7755c0dbe95ccbb71f85c677297f687bf3884fadefc64f1d0fdce51aR22)
+
+Do **not** include `from_presets()` in this PR.
+
+##### Validation Colab
+
+- Load original model weights
+- Manually set weights in your KerasHub model
+- Compare outputs on sample input for closeness
+
+##### Unit Tests (`xx_backbone_test.py`)
+
+- Check forward pass, output shapes
+- Ensure model can be saved and loaded correctly
+
+---
+
+### Step 3: PR #2 – Data Converter
+
+#### Tokenizer / ImageConverter / AudioConverter
+
+The converter transforms raw input into numerical tensors suitable for 
+preprocessing.
+
+##### Implementation
+
+- **Text**: `XXTokenizer`, subclassing from KerasHub tokenizers
```

Comment:
Probably should link these code files (tokenizer, imageconverter, audioconverter).